content/rs/security/encryption/tls/tls-protocols.md

```diff
@@ -18,49 +18,67 @@ The communications for which you can modify TLS protocols are:
 - Data plane - The TLS configuration for the communication between applications and databases.
 - Discovery service (Sentinel) - The TLS configuration for the [discovery service]({{<relref "/rs/databases/durability-ha/discovery-service.md">}}).
 
-You can configure the TLS protocols with the `rladmin` commands shown here or with the REST API.
+You can configure TLS protocols with the [Cluster Manager UI](#edit-tls-ui), [`rladmin`]({{<relref "/rs/references/cli-utilities/rladmin/cluster/config">}}), or the [REST API]({{<relref "/rs/references/rest-api/requests/cluster#put-cluster">}}).
 
 {{<warning>}}
-After you set the minimum TLS version, Redis Enterprise Software does not accept communications with TLS versions older than the specified version.
+- After you set the minimum TLS version, Redis Enterprise Software does not accept communications with TLS versions older than the specified version.
+
+- If you set TLS 1.3 as the minimum TLS version, clients must support TLS 1.3 to connect to Redis Enterprise.
 {{</warning>}}
 
 TLS support depends on the operating system. You cannot enable support for protocols or versions that aren't supported by the operating system running Redis Enterprise Software.  In addition, updates to the operating system or to Redis Enterprise Software can impact protocol and version support.  
 
-To illustrate, version 6.2.8 of Redis Enterprise Software removed support for TLS 1.0 and TLS 1.1 on Red Hat Enterprise Linux 8 (RHEL 8) because that operating system [does not enable support](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening) for these versions by default.  
-
 If you have trouble enabling specific versions of TLS, verify that they're supported by your operating system and that they're configured correctly.
 
 {{<note>}}
 TLSv1.2 is generally recommended as the minimum TLS version for encrypted communications. Check with your security team to confirm which TLS protocols meet your organization's policies.
 {{</note>}}
 
-### Control plane
+### Edit TLS settings in the UI {#edit-tls-ui}
+
+To configure cipher suites using the Cluster Manager UI:
```

Comment:
This section is not for configuring cipher suites, but for configuring the minimum TLS versions

Reply:
Oops. Nice catch. I will fix this.